frontend/integration-tests/views/crud.view.ts

```diff
@@ -21,7 +21,7 @@ export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
 export const isLoaded = () => browser.wait(until.and(untilNoLoadersPresent, untilLoadingBoxLoaded)).then(() => browser.sleep(1000));
 export const resourceRowsPresent = () => browser.wait(until.presenceOf($('.co-m-resource-icon + a')), 10000);
 
-export const resourceRows = $$('.co-resource-list__item');
+export const resourceRows = $$('.ReactVirtualized__VirtualGrid__innerScrollContainer tr');
```

Comment:
We should avoid the `ReactVirtualized__VirtualGrid__innerScrollContainer` selector. That seems really fragile. We should add a `data-test-` attribute if we need to.

Reply:
Ok... give me just a bit...

Reply:
added `data-test-rows` attribute for this since we are selecting many rows for CRUD scenario tests typically.